test/integration/authorization_test.rb

```diff
@@ -24,7 +24,7 @@ class LoginTest < ActionDispatch::IntegrationTest
   test 'you cannot login as a unregistered user' do
     user = create(:user, :default)
     visit root_path
-    fill_in 'session[email_address]', with: 'bob@turing.io'
+    fill_in 'session[username]', with: 'bob'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.